src/services/DFGVariable.jl

```diff
@@ -112,15 +113,11 @@ function ==(a::VariableNodeData,b::VariableNodeData, nt::Symbol=:var)
 end
 
 """
-    $(SIGNATURES)
-Equality check for VariableEstimate.
+    ==(x::T, y::T) where T <: AbstractPointParametricEst
```

Comment:
missed this yesterday sorry, just want to make sure the `import Base: ==` is there, not seeing in the PR.

Reply:
this is the closest I can find:
https://github.com/JuliaRobotics/DistributedFactorGraphs.jl/blob/9afd7a380cc5d124c74bc3a1a466202cd5370fe2/src/LightDFG/FactorGraphs/FactorGraphs.jl#L5

Reply:
Yep, top of the file:
```
import Base: ==, convert
```